portal/component-tests/admin/user-edit.component-test.js

```diff
@@ -0,0 +1,90 @@
+const pageRenderer = require('../pageRenderer');
+
+const page = 'admin/user-edit.njk';
+const render = pageRenderer(page);
+
+describe(page, () => {
+  const adminUser = {
+    timezone: 'Europe/London',
+    roles: ['admin'],
+  };
+  const newUser = { roles: [] };
+
+  const banks = [{
+    id: 1,
+    name: 'Bank 1',
+  }, {
+    id: 2,
+    name: 'Bank 2',
+  }, {
+    id: 3,
+    name: 'Bank 3',
+  }];
+
+  const roles = [{
+    roleName: 'Maker/Checker',
```

Comment:
Will these roles be removed on subsequent tickets?

Reply:
Yep - we have a separate sub-task for this :)